### ISSUE TITLE
Aortitis

### ISSUE THREAD

**Issue by caufieldjh** [obophenotype/human-phenotype-ontology#11448]
**Preferred term label:**
Aortitis

**Synonyms**
aortic inflammation

**Definition (free text, please give PubMed ID)**
Aortitis is the all-encompassing term ascribed to inflammation of the aorta. (PMID:18541754)

The inflammation of aortic wall, named aortitis, is a rare condition that can be caused by a number of pathologies, mainly inflammatory or infectious in nature. (PMID:29489984)

**Parent term (use hpo.jax.org/app)**
Increased inflammatory response (HP:0012649)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
IgG4-related aortitis (ORPHA:449400)
Takayasu arteritis (ORPHA:3287)
Giant cell arteritis (ORPHA:397)
Cogan syndrome (ORPHA:1467)

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-5705-7831

Note: this term exists in MONDO (MONDO:0006656) and in MP (MP:0010139)

**Comment by pnrobinson** [comment]
@caufieldjh  thanks!
Adding new term: 
- ID: HP:6001461
- label: Aortitis
- parents: [Abnormal aortic morphology (HP:0001679)](https://hpo.jax.org/app/browse/term/HP:0001679)
- pmid: PMID:27546992|PMID:33593995
- def: Aortitis refers to inflammatory changes to the wall of the aorta and incorporates both infectious and non-infectious etiologies.
- comment: The term aortitis encompasses inflammatory changes to the aortic wall from varying etiologies that are associated with certain specific histopathological lesions. The inflammatory response may be limited to the aortic wall or accompanied by changes directly adjacent to the vessel (e.g. retroperitoneal fibrosis) or other vessels (e.g., giant cell arteritis). A distinction is made between infectious and non-infectious aortitis. Takayasu arteritis and giant cell arteritis are the most common causes of non-infectious aortitis.
- orcid: orcid:0000-0001-5705-7831
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11448
- synonyms:
